src/ray/gcs/gcs_server/gcs_kv_manager.h

```diff
@@ -53,6 +63,17 @@ class InternalKVInterface {
                    bool overwrite,
                    std::function<void(bool)> callback) = 0;
 
+  /// Associate a key with the specified value.
+  ///
+  /// \param ns The namespace of the key.
+  /// \param key The key for the pair.
```

Comment:
```suggestion
  /// \param key The keys for the pair.
   /// \param value The values for the pair.
```
I assume this is just copied / pasted from Put?